napari/layers/image/image.py

```diff
@@ -176,6 +182,7 @@ def __init__(
         gamma=1,
         interpolation='nearest',
         rendering='mip',
+        mode='volume',
```

Comment:
See other comment about avoiding mode, consider `planar=False` instead if we'll only ever have two modes for foreseeable future